keyboards/takashicompany/spreadwriter/readme.md

```diff
@@ -0,0 +1,35 @@
+# takashicompany/spreadwriter
+
+![takashicompany/spreadwriter](https://i.imgur.com/mMAzyvU.jpg)
```

Comment:
This uses a smaller resolution image, which is more friendly to the github readme rendering. 
```suggestion
![takashicompany/spreadwriter](https://i.imgur.com/mMAzyvUh.jpg)
```

Reply:
Oh. Thanks!